### PR TITLE
test workflow

### PR DESCRIPTION
**Pull request by nuwantha-calcey**

### CODE REVIEW

File: .github/workflows/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
+name: "Pull Request Labeler"
+
+on:
+  pull_request:
+    types:
+      - opened
+      - review_requested
+  pull_request_review:
```
Comment (NuwanthaSandakelum) [inline .github/workflows/labeler.yml:8]: adsfsgdsdgdf

**Review by NuwanthaSandakelum — CHANGES_REQUESTED** [review]

File: .github/workflows/labeler.yml (hunk 1 above)
Comment (nuwantha-calcey) [reply .github/workflows/labeler.yml:8]: done

Comment (nuwantha-calcey) [reply .github/workflows/labeler.yml:8]: hi

File: .github/workflows/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+jobs:
+  dump_contexts_to_log:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Dump GitHub context
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+        run: echo "$GITHUB_CONTEXT"
+
+  label-file-changes:
```
Comment (NuwanthaSandakelum) [inline .github/workflows/labeler.yml:25]: aiyoooo

**Review by NuwanthaSandakelum — CHANGES_REQUESTED** [review]

File: .github/workflows/labeler.yml (hunk 2 above)
Comment (nuwantha-calcey) [reply .github/workflows/labeler.yml:25]: ai

**Review by NuwanthaSandakelum — APPROVED** [review]
LGTM

**Comment by nuwantha-calcey** [comment]
nice work

**Pull request merged**